src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -2,109 +2,50 @@ import React, { useState, useEffect } from 'react';
 
 import PendingIncident from './PendingIncident';
 
-import './AdminDashboard.scss';
+import { useIncidents } from '../../hooks/legacy/useIncidents';
+
+import { falsiesRemoved } from '../incidents/IncidentFilter';
 
-// Mock data to represent incoming twitter data from the database that will be reviewed by the admin
-
-const dummyData = [
-  {
-    categories: [
-      'explosive',
-      'less-lethal',
-      'projectile',
-      'rubber-bullet',
-      'shoot',
-    ],
-    city: 'San Jose',
-    date: '2020-05-29T05:00:00.000Z',
-    desc:
-      'desc A protester filming receives water bottles from a car and begins to distribute them',
-    empty_hand_hard: false,
-    empty_hand_soft: false,
-    incident_id: 'ca-sanjose-5',
-    lat: 37.3353,
-    less_lethal_methods: true,
-    lethal_force: false,
-    long: -121.889,
-    src: ['https://www.youtube.com/watch?v=89mUHzu3480'],
-    state: 'California',
-    title: 'title Man struck by rubber bullet and explosive device',
-    uncategorized: false,
-  },
-  {
-    categories: ['explosive', 'shoot'],
-    city: 'Chicago',
-    date: '2020-05-29T05:00:00.000Z',
-    desc:
-      'A protester filming receives water bottles from a car and begins to distribute them',
-    empty_hand_hard: false,
-    empty_hand_soft: false,
-    incident_id: 'il-chicago-6',
-    lat: 37.3353,
-    less_lethal_methods: true,
-    lethal_force: false,
-    long: -121.889,
-    src: ['https://www.youtube.com/watch?v=89mUHzu3480'],
-    state: 'Illinois',
-    title: 'Man struck by weapon',
-    uncategorized: false,
-  },
-  {
-    categories: ['less-lethal', 'projectile', 'rubber-bullet'],
-    city: 'New York City',
-    date: '2020-06-29T05:00:00.000Z',
-    desc:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt',
-    empty_hand_hard: false,
-    empty_hand_soft: true,
-    incident_id: 'ny-nyc-5',
-    lat: 37.3353,
-    less_lethal_methods: true,
-    lethal_force: false,
-    long: -121.889,
-    src: ['https://www.youtube.com/watch?v=89mUHzu3480'],
-    state: 'New York',
-    title:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt',
-    uncategorized: false,
-  },
-  {
-    categories: ['explosive', 'projectile'],
-    city: 'Los Angeles',
-    date: '2020-07-07T05:00:00.000Z',
-    desc:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt',
-    empty_hand_hard: true,
-    empty_hand_soft: false,
-    incident_id: 'ca-losangeles-5',
-    lat: 37.3353,
-    less_lethal_methods: true,
-    lethal_force: false,
-    long: -121.889,
-    src: ['https://www.youtube.com/watch?v=89mUHzu3480'],
-    state: 'California',
-    title:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt',
-    uncategorized: false,
-  },
-];
+import './AdminDashboard.scss';
```

Comment:
I do think this is supposed to be done differently. This .scss file should be imported into the index.scss file, rather than the component. Then run `sass --watch ./src/index.scss ./src/index.css` (from the root directory, if you use those exact file paths) in terminal, which tells sass to translate our .scss files into the main index.css one, which is what is imported into index.js and applies the styles across the site. I can hop on a zoom and walk through if that would help!